pkg/networkmanager/interfaces.js

```diff
@@ -400,8 +400,8 @@ export function NetworkManagerModel() {
     var watch;
 
     self.preinit.then(() => {
-        subscription = client.subscribe({ }, signal_emitted);
-        watch = client.watch({ });
+        subscription = client.subscribe({ path_namespace: "/org/freedesktop/NetworkManager" }, signal_emitted);
+        watch = client.watch({ path_namespace: "/org/freedesktop/NetworkManager" });
```

Comment:
Is that still necessary after #15617? It certainly can't hurt, but does it still make a difference now?

Reply:
Not necessary anymore, removed.